src/lib/zkapp.ts

```diff
@@ -611,6 +611,50 @@ class SmartContract {
     party.body.events = Events.pushEvent(party.body.events, eventFields);
   }
 
+  fetchEvents(
```

Comment:
I think we should give this an `async` signature. `Mina.fetchEvents` as well. In the local blockchain case, its not really async, but in the remote blockchain case it is, so we need `async` to cover both cases